runtime/posix/src/main/java/org/qbicc/runtime/posix/SysStat.java

```diff
@@ -71,8 +73,12 @@
     public static final mode_t S_IWOTH = constant();
     public static final mode_t S_IXOTH = constant();
 
+    @name(value = "stat$INODE64", when = Build.Target.IsMacOs.class)
     public static native c_int stat(const_char_ptr pathName, struct_stat_ptr statBuf);
+    @name(value = "fstat$INODE64", when = Build.Target.IsMacOs.class)
     public static native c_int fstat(c_int fd, struct_stat_ptr statBuf);
+    @name(value = "lstat$INODE64", when = Build.Target.IsMacOs.class)
     public static native c_int lstat(const_char_ptr pathName, struct_stat_ptr statBuf);
+    @name(value = "fsstat$INODE64", when = Build.Target.IsMacOs.class)
```

Comment:
Is this meant meant to be:
```suggestion
    @name(value = "fstatat$INODE64", when = Build.Target.IsMacOs.class)
```
?